reactioncommerce/reaction
# (fix): orders date filter not appearing

Fixes #3402 

This PR updates the `CalendarPicker` component as a recent update of `react-dates` introduced some major braking changed

## Highlights of this change

- React dates must be Initialized react dates before importing
- CSS classNames and some properties re-worked to be compatible `react-dates`
- Fixes RTL support for the calendar picker.

## To Test

With a copy of this branch and a fresh `meteor npm install`...

- Open the orders dashboard and attempt to use the date filters. It should work as it used to.
- Add some orders to test if the date filtering still works.
- Switch to an RTL language, see that the date picker is now RTL enabled
